### PR TITLE
Add release drafter

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name-template: v$NEXT_MINOR_VERSION 🌈
+tag-template: configuration-as-code-$NEXT_MINOR_VERSION
```
Comment (casz) [inline release-drafter.yml]: copypasta! :see_no_evil:

File: release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - title: 👻 Maintenance
+    label: chore
+  - title: 📝 Documentation updates
+    label: documentation
+  - title: 🚦 Tests
+    label: test
+
+template: |
+  ## Changes
+  $CHANGES
```
Comment (casz) [inline release-drafter.yml]: I am pretty sure you want a space between headers :sweat:

File: release-drafter.yml (hunk 1 above)
Comment (timja) [reply release-drafter.yml]: 🙈

File: release-drafter.yml (hunk 2 above)
Comment (timja) [reply release-drafter.yml]: that what you meant?

Comment (casz) [reply release-drafter.yml]: yup :sweat_smile:

Comment (casz) [reply release-drafter.yml]: I updated my first comment :neutral_face:

**Pull request merged**